aibolit/patterns/send_null/send_null.py

```diff
@@ -9,59 +9,37 @@ class SendNull:
     def __init__(self):
         pass
 
-    def __find_position_recursively(self, node):
-        if not hasattr(node, 'children'):
-            return
-        else:
-            for i in node.children:
-                if not isinstance(i, list):
-                    if hasattr(i, '_position'):
-                        if i.position:
-                            return i._position.line
-                        else:
-                            for j in i.children:
-                                position = self.__find_position_recursively(j)
-                                if position:
-                                    return position
+    def __is_null(self, val):
```

Comment:
I hardly can find this function neither readable nor maintainable. Hope you too.

Reply:
@lyriccoder I think you can write it more elegant